### PR TITLE
docs: documentation on implementation

### PR DESCRIPTION
**Pull request by andy-haynes**
# Description

Add documentation on how a Wallet Selector wallet integration can implement account import. This should be merged once the encryption method is brought over so that these docs can reference it.

# Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

# Type of change.

- [ ] FIX - a PR of this type patches a bug.
- [ ] FEATURE - a PR of this type introduces a new feature.
- [ ] BUILD - a PR of this type introduces build changes.
- [ ] CI - a PR of this type introduces CI changes.
- [x] DOCS - a PR of this type introduces DOCS improvement.
- [ ] STYLE - a PR of this type introduces style changes.
- [ ] REFACTOR - a PR of this type introduces refactoring.
- [ ] PERFORMANCE - a PR of this type introduces performance changes.
- [ ] TEST - a PR of this type adds more tests.
- [ ] CHORE - a PR introduces other changes than the specified above.

### CODE REVIEW

**Comment by kujtimprenkuSQA** [comment]
Hi, @andy-haynes I know this is still a draft PR just wanted to add some pointers here:

In the docs here: [ExportSelectorOptions](https://github.com/near/wallet-selector/tree/account-export-documentation/packages/account-export#options) we need to add the `onComplete` too since is part of the interface:
https://github.com/near/wallet-selector/blob/account-export-documentation/packages/account-export/src/lib/index.types.ts#L8

It might be also good to add an API Reference section as in the `modal-ui` package: https://github.com/near/wallet-selector/tree/main/packages/modal-ui#api-reference right now this API looks the same but in the future, it might change for either of these packages.

**Comment by hcho112** [comment]
I have received an advice regards to security experts in the team. For the wallet selector, we should encourage receiving wallet to execute `history.replaceState()` after receiving the hash data to make sure history is not stored.

Can we mention this on the doc?

File: packages/account-export/INTEGRATION.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  /* this URL must exist and parse the fragment data */
+  return `https://awesome-near-wallet.xyz/${this.state.networkId}`;
+}
+```
+
+The Wallet Selector will use this base URL and append encrypted data to it before redirecting there, so the final URL would be in the form of `"https://awesome-near-wallet.xyz/mainnet#xyz...abc"`
+
+:warning:
+**Once the user is redirected, the destination site should call `history.replaceState()` to prevent user data from being unnecessarily exposed in the browser.**
+:warning:
```
Comment (hcho112) [inline packages/account-export/INTEGRATION.md:39]: this warning label here shown as:
image

is it mean to be `**warning**` ?

Comment (andy-haynes) [reply packages/account-export/INTEGRATION.md:39]: `:warning:` just shows ⚠️, just something to grab attention. You can see it formatted here: https://github.com/near/wallet-selector/blob/e50370145ee05f3643f8d4488c47f2f09018678f/packages/account-export/INTEGRATION.md

Comment (hcho112) [reply packages/account-export/INTEGRATION.md:39]: my apologies, thought markdowns works similarly on online readme editor tools..

**Review by hcho112 — DISMISSED** [review]
LGTM, good description specially with use of `useUrlAccountImport`.

**Pull request merged**